lib/msal-core/src/UserAgentApplication.ts

```diff
@@ -635,9 +635,9 @@ export class UserAgentApplication {
                 request.correlationId,
             );
 
-            // populate QueryParameters (sid/login_hint/domain_hint) and any other extraQueryParameters set by the developer
+            // populate QueryParameters (sid/login_hint) and any other extraQueryParameters set by the developer
             if (ServerRequestParameters.isSSOParam(request) || account) {
-                serverAuthenticationRequest.populateQueryParams(account, request);
+                serverAuthenticationRequest.populateQueryParams(account, request, true);
```

Comment:
Missing parameter. The true here would be mapped to adalIdTokenObject instead of silentCall

Reply:
Same as above